IFC4x3/Properties/e/EC_0JOFocHPD1Rxfnqm5NX9ca/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EC_0JOFocHPD1Rxfnqm5NX9ca" Name="EC" UniqueId="1360fca6-4593-416f-ba71-d301578499a4" PrimaryDataType="IfcReal" />
```

Comment:
Propose "ElectricalConductivity" and datatype `IfcElectricConductanceMeasure`.

Reply:
Done.